Makefile

```diff
@@ -111,28 +111,44 @@ ifndef VERSION
 VERSION=1.3.0
 endif
 
+ifndef PACKAGE
+PACKAGE=tremulous-grangerhub
+endif
+
 ifndef CLIENTBIN
 CLIENTBIN=tremulous
 endif
 
+ifndef CLIENTBINSH
+CLIENTBINSH=$(CLIENTBIN).sh
+endif
+
 ifndef SERVERBIN
 SERVERBIN=tremded
 endif
 
+ifndef SERVERBINSH
+SERVERBINSH=$(SERVERBIN).sh
+endif
+
 ifndef BASEGAME
 BASEGAME=gpp
 endif
 
 BASEGAME_CFLAGS=-I../../${MOUNT_DIR}
 
 ifndef COPYDIR
-COPYDIR="/usr/local/games/tremulous"
+COPYDIR="/opt/$(PACKAGE)"
```

Comment:
Why `/opt`?

Reply:
I was following what was done in the Arch Linux packaging. Since Tremulous binaries need to stay in the data folder, it doesn't correspond to the usual Linux way of storing stuff under PREFIX/share/APPNAME format. But I'm not confident in that. There used to be a separate tremulous-data package. Not sure which approach is better.

Reply:
Also, /usr/local/games seems to be an unused location in modern distros. Again, I'm not confident about this.

Reply:
But /opt is usually not used anymore by most software. You can use something like `$PREFIX/share/tremulous` (/usr/local/share/tremulous if PREFIX is /usr/local).

Take a look how we package it in Fedora:
https://koji.fedoraproject.org/koji/rpminfo?fileOrder=name&rpmID=37498623&buildrootOrder=-id&buildrootStart=0#filelist
https://koji.fedoraproject.org/koji/rpminfo?fileOrder=name&rpmID=37497768&buildrootOrder=-id&buildrootStart=0#filelist